# genome-superscaffolds
Scripts for generating superscaffolds using gene synteny

These scripts take files generated by reciprocally blasting two flanking genomes 
against a target genome to establish gene correspondence, then consider the scaffolds
in the target genome. Where scaffolds in both flanking genomes cover a junction in
the target genome the two target scaffolds are linked as putatively contiguous.

The paired contiguous scaffolds in the target are then daisy chained into super-
scaffolds, as an aid to physical mapping.

The script synteny_pairwise.cgi generates interim 2-value superscaffolds based on 
a single flanking taxon.

The script synteny_combine.cgi combines the results of pairwise comparisons to 
yield a table of putative junctions between target taxon scaffolds together 
with the support for these junctions drawn from consensus across multiple
flanking taxa.

The script synteny_chain.cgi daisy-chains the interim 2-value super-scaffolds into
a final set of putative superscaffolds.

It is sensible to choose flanking taxa that span the target taxon phylogenetically
if possible.
